### PR TITLE
Fixing the build file

### PR DESCRIPTION
**Pull request by Aisuko**
Signed-off-by: aisuko <urakiny@gmail.com>

**Description**

We did not push any Docker image to DockerHub and we should fix the issue.

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ RUN go mod download
 # Copy the go source
 COPY . .
 # Build
-RUN CGO_ENABLED=1 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o meshery-meshsync meshsync.go
+RUN CGO_ENABLED=1 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o meshery-meshsync main.go
```
Comment (leecalcote) [inline Dockerfile:13]: Thanks for this PR, @Aisuko. I’m probably confused - isn’t the name of the file to compile `meshsync.go`? https://github.com/layer5io/meshsync/tree/master/meshsync

Comment (leecalcote) [reply Dockerfile:13]: Yes, I am confused. Abishek pointed out https://github.com/layer5io/meshsync/blob/master/main.go to me now.

Comment (Aisuko) [reply Dockerfile:13]: Yep, we want to compile our file to `meshery-meshsync`.

```
usage: go build [-o output] [-i] [build flags] [packages]

```

**Comment by kumarabd** [comment]
Thanks for this @Aisuko

**Review by kumarabd — APPROVED** [review]

**Pull request merged**